recipes/mmvec/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
You don't need this file here if it's in the sdist

Reply:
removed

Reply:
The file is still there

Reply:
misread. The file is now gone.